WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -39,7 +39,10 @@ + (NSArray *)routes
     [[FBRoute POST:@"/wda/homescreen"].withoutSession respondWithTarget:self action:@selector(handleHomescreenCommand:)],
     [[FBRoute POST:@"/wda/deactivateApp"] respondWithTarget:self action:@selector(handleDeactivateAppCommand:)],
     [[FBRoute POST:@"/wda/keyboard/dismiss"] respondWithTarget:self action:@selector(handleDismissKeyboardCommand:)],
-    [[FBRoute GET:@"/lock"].withoutSession respondWithTarget:self action:@selector(handleLock:)],
+    [[FBRoute POST:@"/wda/lock"].withoutSession respondWithTarget:self action:@selector(handleLock:)],
```

Comment:
I think this is actually a `PUT` since the end state is always a locked screen, I'm not considering the case of it failing since it is irrelevant in http methods. The same applies to unlock

Reply:
I'd rather follow the general pattern.